README.md

```diff
@@ -352,476 +339,1080 @@ You can use the aws module to audit AWS resources, launch autoscaling groups in
+Valid values: 'SSLv3', 'TLSv1', 'TLSv1.1', 'TLSv1.2'.
 
 ##### `tags`
-*Optional* The tags for the security group. Accepts a 'key => value' hash of tags.
 
-##### `description`
-*Required* A short description of the group. This parameter is set at creation only; it is not affected by updates.
+Optional.
+ 
+The tags for the distribution. 
 
-##### `vpc`
-*Optional* The VPC to which the group should be associated. This parameter is set at creation only; it is not affected by updates. Accepts the value of the Name tag for the VPC.
+Accepts a 'key => value' hash of tags. Excludes 'Name' tag.
 
+####Type: ec2_instance
 
-#### Type: elb_loadbalancer
+#####`ensure`
+
+Specifies the basic state of the resource. 
+
+Valid values are: 'present', 'absent', 'running', 'stopped'.
 
 #####`name`
-*Required* The name of the load balancer. This is the value of the AWS Name tag.
 
-#####`region`
-*Required* The region in which to launch the load balancer. For valid values, see [AWS Regions](http://docs.aws.amazon.com/general/latest/gr/rande.html#ec2_region).
+Required.
+ 
+The name of the instance. This is the value of the AWS Name tag.
 
-#####`listeners`
-*Required* The ports and protocols the load balancer listens to.  Accepts an
-array of the following values:
-  * protocol
-  * load_balancer_port
-  * instance_protocol
-  * instance_port
-  * ssl_certificate_id (required if protocol is HTTPS)
-  * policy_names (optional array of policy name strings for HTTPS)
+#####`security_groups`
 
-#####`health_check`
-The configuration for an ELB health check used to determine the health of the
-back- end instances.  Accepts a hash with the following keys:
-  * healthy_threshold
-  * interval
-  * target
-  * timeout
-  * unhealthy_threshold
+Optional. 
+
+The security groups with which to associate the instance. 
+
+Accepts an array of security group names.
 
 #####`tags`
-*Optional* The tags for the load balancer. This parameter is set at creation only; it is not affected by updates. Accepts a 'key => value' hash of tags.
 
-#####`subnets`
-*Optional* The subnet in which the load balancer should be launched. Accepts an array of subnet names, i.e., the Name tags on the subnets. You can only set one of `availability_zones` or `subnets`.
+Optional. 
 
-#####`security_groups`
-*Optional* The security groups to associate with the load balancer (VPC only). Accepts an array of security group names, i.e., the Name tag on the security groups.
+The tags for the instance. 
 
-#####`availability_zones`
-*Optional* The availability zones in which to launch the load balancer. This parameter is set at creation only; it is not affected by updates. Accepts an array on availability zone codes. For valid availability zone codes, see [AWS Regions and Availability Zones](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html). You can only set one of `availability_zones` or `subnets`.
+Accepts a 'key => value' hash of tags.
```

Comment:
key => value isn't a string value, so it doesn't need to be in single quotes. TBH I'm not sure it needs to be marked at all, because it's a description of the hash. And I think it's probably OK to translate, even.